src/components/select/select.scss

```diff
@@ -67,6 +67,7 @@ label {
     letter-spacing: 0;
     line-height: 1.4;
     background-color: var(--input-bg);
+    width: 98%;
```

Comment:
Why 98%?

Reply:
Select component's input field's click area goes beyond the component width. It causes design breakage

Reply:
Add `box-sizing:border-box` with `width:100%`. See if that works.